test/controllers/RegistrationSentControllerSpec.scala

```diff
@@ -91,24 +95,24 @@ class RegistrationSentControllerSpec extends SpecBase with ImplicitDateFormatter
       val result = controller.onPageLoad()(fakeRequest)
 
       status(result) mustEqual OK
-      contentAsString(result) mustEqual view(dayToString(inject[TimeMachine].now().plusDays(28)),
+      contentAsString(result) mustEqual view(dayToString(inject[TimeMachine].now().plusDays(daysToAdd)),
         dayToString(inject[TimeMachine].now(), dayOfWeek = false),
-        "123456789", emailOrPost = true, noEmailOrPost = true,
+        acknowledgementReferenceNo, emailOrPost = true, noEmailOrPost = true,
         Seq("requiredDocuments.governingDocumentName.answerTrue"), None)(fakeRequest, messages, frontendAppConfig).toString
       verify(mockUserAnswerService, times(1)).get(any())(any(), any())
     }
 
     "return OK and the correct view for a GET when noEmailPost enabled" in {
 
       when(mockUserAnswerService.get(any())(any(), any())).thenReturn(Future.successful(Some(emptyUserAnswers
-        .set(AcknowledgementReferencePage, "123456789")
+        .set(AcknowledgementReferencePage, acknowledgementReferenceNo)
         .flatMap(_.set(ApplicationSubmissionDatePage, inject[TimeMachine].now()))
         .success.value)))
 
       val result = controller.onPageLoad()(fakeRequest)
 
       status(result) mustEqual OK
-      contentAsString(result) mustEqual view(dayToString(inject[TimeMachine].now().plusDays(28)),
+      contentAsString(result) mustEqual view(dayToString(inject[TimeMachine].now().plusDays(daysToAdd)),
         dayToString(inject[TimeMachine].now(), dayOfWeek = false),
         "123456789", emailOrPost = false, noEmailOrPost = true,
```

Comment:
stray `acknowledgementReferenceNo`?

Reply:
this is a dummy acknowledgeReferenceNo for test, so no issues